source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -291,8 +359,44 @@ content: |
              status: 0,
              masterKey: { provider: 'local' }
            }
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+
+           from pprint import pprint
+           connection_string = "mongodb://localhost:27017"
+           key_vault_namespace = "encryption.__keyVault"
+
+
+           def verify_data_key_created(fle_opts, client, client_encryption,  data_key):
+               db = client["encryption"]
+               collection = db["__keyVault"]
+               key = collection.find_one({"_id": data_key})
+               pprint(key)
+               client_encryption.close()
+               client.close()
 
 
+           # pass in fle_opts, client, client_encryption, data_key variables created in previous section
+           verify_data_key_created(fle_opts, client, client_encryption, data_key)
+
+
+        This code example should print a retrieved document that resembles the
+        following:
+
+        .. code-block:: none
+
+           {
+             "_id": UUID('1e83d013-d873-47df-abb1-e57898a72d4c'),
+             "keyMaterial":  b'\x96#\xeb\xa1xKA\xa7GM\xef\x08\xc04\'gD\x96\x9c\xa4\xd3?\xe3Db0H\xbb\x86\xa5\xc2\x1f\x14\x0f\xb8\xb8\x8a\x9d\xc9\xae\xa1g\xaf\xeb\x8b\x99\xb4b"\xc0\xe8e\x07\x1b.\xeet\xf5<%\xfd\x06Y\x15o=3Yk\x9fue\xd8V#X\xc1IB\xf5\xc9+\x95\xb1\x9c\xc0\x08U?\xaf\xb1U\xc6\x84\x89\x9b\xdc\x98\xc9~\xb2\xbd\xf6\\\xa2y\x08\xdf\x8f\xa1\x03\t9\xe7_+J>_H\xb4\x97up\x93Sc\x88\x0fG-+\x86\x95\x9e\xc2\x8es\x9e\xcb%%lVQ\xa2\xf1\xe3W\x83\x10]\xc9\x1fm\x7f\xbc\xbf\xd2d',
+             "creationDate":  datetime.datetime(2019, 9, 30, 20, 43, 10, 951000),
```

Comment:
Nitpick: looks like there's an extra space here and in the line above. Is this verbatim what gets printed?

Reply:
Not verbatim what gets printed, the actual log is quite a bit uglier even w/ pprint (pretty print). The real thing looks more like this:

```
{u'_id': UUID('46ade0a2-fe52-4dec-af7d-ef3963a66414'),
 u'creationDate': datetime.datetime(2019, 10, 15, 13, 46, 26, 126000),
 u'keyMaterial': Binary('\xb8\x1c\xf4_\x18\t\xf3\xa8\xe1\x96p\x87|\xdb\x0e4G:\xf3/\x03\x04c\xfb\xb0\x19\xe6\xadPg\xd2G\xcaH\x8f\xe0\x0fsAG\x06\x9c\xe8\xc0\xbf}\x8f{\xa2\xcd\xce\xb2Z<\xc9mR\xda\xd1&\x8a\xc5aeTzx\xe9\xcf\xad&\xb5y\x89\xa6Cn\xfc6\x87\xc6\x1c\x8b\xa5\x16\xfb\xa1\xf97xx\x91\x0b1\x03\x82m\xbc\xe0\x85\xb1<3eO\r\xba\xc6\x8c\x11\x8b\x0f0lK\xa3\xae\x91\x94^\xdb\x9d\x89T\xee\x898}:^X\x82+\xbaK\x9f\x96\x18\x8f\xb5\xea\x18\xf5\x15 ~\x96H\x99\xab\x06D\xc9\x19\xa6\xbb\xe4\x8e\x7f\xa2', 0),
 u'masterKey': {u'provider': u'local'},
 u'status': 0,
 u'updateDate': datetime.datetime(2019, 10, 15, 13, 46, 26, 126000)}
```


---- On the bright side, I just pushed a fix for the extra whitespace

Reply:
Ah ok, I just meant the extra space as I would not have expected pretty print to add one.